packages/api/src/task/scheduler.ts

```diff
@@ -273,6 +273,27 @@ export default class TaskScheduler {
     }
   }
 
+  async deleteAsset(asset: string | Asset) {
+    if (typeof asset === "string") {
+      asset = await db.asset.get(asset);
+    }
+    await db.asset.delete(asset.id);
+    const timestamp = asset.status.updatedAt;
+    await this.queue.publishWebhook("events.asset.deleted", {
+      type: "webhook_event",
+      id: uuid(),
+      timestamp,
```

Comment:
The webhook timestamp should be the time on which the respective event happened, and the asset `updatedAt` is not (and doesn't need to) be updated when it is deleted.

IOW:
```suggestion
    await this.queue.publishWebhook("events.asset.deleted", {
      type: "webhook_event",
      id: uuid(),
      timestamp: Date.now(),
```

Reply:
Right! makes sense